### ISSUE TITLE
PlatformTabController uses the wrong underlying controller

### ISSUE THREAD

**Issue by phantomato** [aqwert/flutter_platform_widgets#126]
PlatformTabScaffold crashes in Android context.

```
void _init(BuildContext context) {
    if (isMaterial(context)) {
      if (_materialController == null) {
        int useIndex = ios?.initialIndex ?? initialIndex;
        if (_cupertinoController != null) {
          useIndex = _materialController.index;

          _cupertinoController.removeListener(_listener);
          _cupertinoController.dispose();
          _cupertinoController = null;
        }
        _materialController = MaterialTabController(
          initialIndex: useIndex,
        );
      }
    }
    if (isCupertino(context)) {
      if (_cupertinoController == null) {
        int useIndex = ios?.initialIndex ?? initialIndex;
        if (_materialController != null) {
          useIndex = _materialController.index;

          _materialController.removeListener(_listener);
          _materialController.dispose();
          _materialController = null;
        }

        _cupertinoController = CupertinoTabController(
          initialIndex: useIndex,
        );
      }
    }
  }
```

On line 451: `useIndex = _materialController.index;` looks like it should use `_cupertinoController` instead.

**Comment by aqwert** [comment]
Correct. I am pushing out a fix very soon. Thanks

**Comment by aqwert** [comment]
This should be resolved with v 0.32.1
